tensorflow/core/api_def/base_api/api_def_BroadcastTo.pbtxt

```diff
@@ -18,24 +18,30 @@ END
 A Tensor.
 END
   }
-  summary: "Broadcast an array for a compatible shape."
+  summary: "Broadcast an array to a compatible shape."
   description: <<END
 Broadcasting is the process of making arrays to have compatible shapes
 for arithmetic operations. Two shapes are compatible if for each
 dimension pair they are either equal or one of them is one. When trying
-to broadcast a Tensor to a shape, it starts with the trailing dimensions,
+to broadcast a Tensor to a shape, this op starts with the trailing dimensions
 and works its way forward.
 
 For example,
+
 ```
->>> x = tf.constant([1, 2, 3])
+>>> x = tf.constant([1, 2, 3], shape=[1, 3])
```

Comment:
Probably not necessary, though it is fine either way.

Reply:
I added `shape=[1, 3]` here because the first sentence in the next paragraph says the shape of `x` is `[1, 3]`. The original expression `tf.constant([1, 2, 3])` produces a tensor with shape `[3]`. I'm also fine either way :)